eslint.config.js

```diff
@@ -45,6 +43,12 @@ export default [
       'jsdoc/check-indentation': 'off'
     }
   },
+  {
+    files: ['**/packages/**/client/**/*.ts'],
+    rules: {
+      '@typescript-eslint/consistent-type-definitions': 'off'
+    }
+  },
```

Comment:
Do we still need this as we already ignored all clients above?

Reply:
🤦🏽‍♀️